gz_ros2_control_demos/CMakeLists.txt

```diff
@@ -47,6 +48,14 @@ target_link_libraries(example_effort PUBLIC
   rclcpp_action::rclcpp_action
 )
 
+# use the same example_position.cpp for example_ft_sensor
+add_executable(example_ft_sensor examples/example_position.cpp)
+target_link_libraries(example_ft_sensor PUBLIC
+  ${control_msgs_TARGETS}
+  rclcpp::rclcpp
+  rclcpp_action::rclcpp_action
+)
+
```

Comment:
```suggestion
```
we can just use the example_position directly. I added the other "alias" executables only for legacy reasons.